src/dxtbx/model/detector_helpers.py

```diff
@@ -294,15 +300,10 @@ def set_detector_distance(detector, distance):
     detector[0].set_frame(fast_axis, slow_axis, origin)
 
 
-def get_detector_projection_2d_axes(detector):
-
-    """Project panel origins, fast and slow axes onto the best-fitting 2D plane.
-
-    :param detector: detector used to generate axes
-    :type detector: Detector
+def get_detector_projection_2d_axes(detector: Detector) -> Tuple[Tuple, Tuple, Tuple]:
```

Comment:
You could define a [type alias](https://docs.python.org/3/library/typing.html#type-aliases) to make the return signature slightly clearer:
```suggestion
Coord2d = Tuple[float, float]

def get_detector_projection_2d_axes(detector: Detector) -> Tuple[Coord2d, Coord2d, Coord2d]:
```